config.toml.example

```diff
@@ -721,6 +721,9 @@ changelog-seen = 2
 # probably don't want to use this.
 #qemu-rootfs = <none> (path)
 
+# Skip building the `std` library for this target.
```

Comment:
Can we note here that this is default enabled for targets containing -none, nvptx, switch, and uefi in their name?

Reply:
Done